mongodb/src/main/scala/akka/stream/alpakka/mongodb/scaladsl/MongoFlow.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.mongodb.scaladsl
+
+import akka.stream.scaladsl.Flow
+import akka.NotUsed
+import org.mongodb.scala.bson.conversions.Bson
+import org.mongodb.scala.model.UpdateOptions
+import org.mongodb.scala.result.{DeleteResult, UpdateResult}
+import org.mongodb.scala.{Document, MongoCollection}
+
+import scala.concurrent.ExecutionContext
+
+object MongoFlow {
+
+  /**
+   * A [[Flow]] that will insert documents into a collection.
+   * @param parallelism number of documents to insert in parallel.
+   * @param collection mongo db collection to insert to.
+   */
+  def insertOne(parallelism: Int, collection: MongoCollection[Document])(
+      implicit executionContext: ExecutionContext
+  ): Flow[Document, Document, NotUsed] =
+    Flow[Document]
+      .mapAsyncUnordered(parallelism)(doc => collection.insertOne(doc).toFuture().map(_ => doc))
```

Comment:
Isn't the unordered part pretty important to document?
Looks like they are inserted in order but the results (out Document) can be unordered.
Wonder if it wouldn't be performant enough to use ordinary `mapAsync` and avoid the confusion that might be caused by the unordered?

Reply:
Fair point. Ill take it out. Maybe in a later PR someone can speed test it and add back unordered variants.